<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Card</class>
 <widget class="QDialog" name="Card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="ListMusic">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>640</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="DeleteWay"/>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonDelete">
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="EtatCard">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>390</y>
     <width>591</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>640</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Haut-Parleur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SpeakerTest">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>640</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tester</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
